package.json

```diff
@@ -25,7 +25,9 @@
   },
   "dependencies": {
     "@actions/core": "^1.6.0",
+    "@actions/github": "^5.0.3",
```

Comment:
This is needed to read `github.context.payload`, which contains the base ref of a PR